### DISCUSSION TITLE
Multi line MsgBox

### DISCUSSION THREAD

**Discussion by YogiYang** [twinbasic/twinbasic#790] [Ideas]
By default MsgBox in VB6 is on one single like and if we want multiple lines we have to add VBCRLF in between each line.

Instead of that can you give a super powered version of MsgBox where user can enter multiple line messages simply by entering comma in between each line.

The new syntax would be something like this:
`MsgBox(<Message1>[,Message2,..,..,MessageN],<Icon + Buttons>,<Title>)`

Sample usage 1:
`MsgBox("Message Line #1","Message Line #2","Message Line #3","","Message Line #4",vbCritical ,"Warning")`

Sample usage 2:
`MyAns = MsgBox("Are you sure you want to delete this record?","Data once deleted cannot be undone",vbCritical  + vbYesNo,"Confirm Delete")`

**Comment by FullValueRider** [comment]
This is a pointless request as it is very simple to build a string interpolator class so that you can do something like 
```
MsgBox Fmt.Txt("{0}{nl}{1}{nl}", "Message Line 1", "Message Line 2"). 
```
See the 'CSharpishStringFormatter' package by Mathieu Guindon that I uploaded.

**Reply by EduardoVB** [reply]
I don't see it as possible, since the MsgBox function already has several parameters.
You could do it only if the Prompt parameter was the last one.
